packages/terra-clinical-result/src/proptypes/valuePropTypes.js

```diff
@@ -1,40 +1,15 @@
 import PropTypes from 'prop-types';
 
-const valueQuantityPropShape = PropTypes.shape({
+const valueQuantityPropType = PropTypes.shape({
   /**
-   *  Numeric Value for an Observation Result
+   *  Value for a single Observation Result.
+   *  can be either single string or array of string values for multi-alpha responses
    */
-  value: PropTypes.number,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
I thought we discussed remove `PropTypes.object` from here. what is the object structure expected here? We need to document the key-value pairs we expect consumers to pass down

Reply:
We discussed removing it but had to add the object back, it's either a string or an array of strings. We should be more specific about this being an array of strings.